pkg/model/api/v1/dashboard/layout.go

```diff
@@ -85,8 +85,9 @@ type GridLayoutDisplay struct {
 }
 
 type GridLayoutSpec struct {
-	Display *GridLayoutDisplay `json:"display,omitempty" yaml:"display,omitempty"`
-	Items   []GridItem         `json:"items" yaml:"items"`
+	Display        *GridLayoutDisplay `json:"display,omitempty" yaml:"display,omitempty"`
+	Items          []GridItem         `json:"items" yaml:"items"`
+	RepeatVariable string             `json:"repeatVariable,omitempty" yaml:"repeatVariable,omitempty"`
```

Comment:
this should be reflected into the go-sdk. Today there is no easy way to use this functionality in the go-sdk